addons/mail/models/mail_activity_mixin.py

```diff
@@ -455,7 +456,7 @@ def activity_feedback(self, act_type_xmlids, user_id=None, feedback=None, attach
             activities.action_feedback(feedback=feedback, attachment_ids=attachment_ids)
         return True
 
-    def activity_unlink(self, act_type_xmlids, user_id=None):
+    def activity_unlink(self, act_type_xmlids, user_id=None, only_automated=True):
```

Comment:
Would also add it to ``activity_feedback`` and ``activity_reschedule`` so that API is consistent :)